RELEASE_PROCEDURE.md

```diff
@@ -9,7 +9,8 @@
    - Increase the version and apply other release-related changes
      - MUST do:
        - Version bumping: In stellargraph/version.py, change version from “X.X.Xb” to “X.X.X”. E.g. version=”0.2.0b” to version=”0.2.0”
-       - Other release-related changes e.g. changelog update
+       - Update Changelog section header to point to specific version tag instead of `HEAD`. Note: this link will be broken until the tag is pushed later.
+       - Remove "Full Changelog" link from Changelog.
```

Comment:
I think this should be the same as the previous line, where `HEAD` is changed to the tag. For instance, https://github.com/stellargraph/stellargraph/blob/develop/CHANGELOG.md#080 tries to link to https://github.com/stellargraph/stellargraph/compare/v0.7.3...v0.8.0 (it uses https://github.com/stellargraph/stellargraph/compare/v0.8.0...v0.7.3 which has the wrong order). I guess we haven't done this for hotfixes.

That is, potentially the previous line can be:

```markdown
- Update Changelog section header and "Full Changelog" link to point to specific version tag instead of `HEAD`. Note: these links will be broken until the tag is pushed later.
```

and

```suggestion
```